tests/datadriven/test_variants.py

```diff
@@ -7,12 +7,12 @@
 
 import os
 import glob
-
```

Comment:
We should keep this space, as we want to separate standard library imports from 3rd party packages --- see the note on imports in STYLE.md